[{
  "comments": [{
    "author": {
      "login": "kayousterhout"
    },
    "body": "can you take the error message as a param to NonDeserializableTask, so the magic string can be defined once and used below on line 137? From a quick read I thought the one on line 137 was coming from the Spark core code (not from the test code)",
    "commit": "ce6bf9abe3d1e97cc8f02de6331b1759630622dd",
    "createdAt": "2017-02-24T01:47:01Z",
    "diffHunk": "@@ -133,6 +122,79 @@ class ExecutorSuite extends SparkFunSuite {\n       }\n     }\n   }\n+\n+  test(\"Gracefully handle error in task deserialization\") {\n+    val conf = new SparkConf\n+    val serializer = new JavaSerializer(conf)\n+    val env = createMockEnv(conf, serializer)\n+    val serializedTask = serializer.newInstance().serialize(new NonDeserializableTask)",
    "line": 118
  }],
  "prId": 16930
}]